docs/development/repositories.md

```diff
@@ -2,18 +2,61 @@
 sidebar_position: 3
 ---
 
-TODO: Write doc
+# Repositories
 
+Our code repositories are hosted on Github. Initially, we created our first repository under the Github user profile [equetzal](https://github.com/equetzal) to take advantage of the Github Student Pack and Github Pro features during her university studies. Subsequently, we established the [huronOS Github Organization](https://github.com/orgs/huronOS/repositories) to house all our new repositories. Eventually, we plan to migrate the first repository, huronOS-build-tools, to the organization. However, being a free organization, we do not have access to certain advanced features on Github. If we receive enough donations in the future, we may consider purchasing Github Enterprise or a similar solution. Alternatively, if we have our own infrastructure, we might explore the option of self-hosting with GitLab.
 
-## Repositories
+## Current Repositories
 
-### huronOS-build-tools
+Below is the current structure of our code repositories:
 
-### website
+### [huronOS-build-tools](https://github.com/equetzal/huronOS-build-tools)
 
-### syslinux
+This is the primary repository containing a collection of scripts that facilitate the construction of huronOS using a Debian installation as the base. It also serves as a quasi-monorepo, encompassing numerous scripts, features, configurations, and software that are utilized by the system but not distributed across multiple repositories. Additionally, this repository includes documentation, ensuring that the documentation stays synchronized with the evolving codebase.
```

Comment:
To avoid redundance I would only write "... this repository includes documentation, ensuring it stays synchronized with the..."